src/common/extract_metadata/config.vsh.yaml

```
@@ -0,0 +1,35 @@
+functionality:
+  name: extract_metadata
+  namespace: common
+  description: Extract the metadata from an h5ad file.
+  argument_groups:
+    - name:  Inputs
+      arguments:
+        - name: --input
+          type: file
+          required: true
+          description: A h5ad file.
```

```suggestion
        - name: --input
          type: file
          required: true
          description: A h5ad file.
        - name: --schema
          type: file
          required: false
          description: An optional schema with which to annotate the output
```

↑ What would you think about adding an optional `schema` so we can add descriptions to the fields that are in the AnnData and also in the schema?

---

ok for me

---

Working on it!

---

Done!